src/Microsoft.SqlTools.ServiceLayer/Utility/FileUtils.cs

```diff
@@ -8,6 +8,46 @@ namespace Microsoft.SqlTools.ServiceLayer.QueryExecution
 {
     internal static class FileUtils
     {
+        internal static string PeekDefinitionTempFolder = Path.GetTempPath() + "mssql_definition"; 
```

Comment:
Since all this code is prefixed with PeekDefinition, I think it would work better to put it in the language service instead of the globally available FileUtils class.